pkg/conduit/config_test.go

```diff
@@ -160,6 +160,7 @@ func TestConfig_Validate(t *testing.T) {
 			validConfig.Log.Level = "info"
 			validConfig.Log.Format = "cli"
 			validConfig.Pipelines.Path = "./pipelines"
+			validConfig.Schema.Type = SchemaTypeInMemory
```

Comment:
I _think_ the above lines can be replaced with:
```go
validConfig := DefaultConfig()
validConfig.DB.Postgres.ConnectionString = "postgres://user:pass@localhost:5432/mydb?sslmode=disable"
```

Reply:
@hariso good point! Much easier to maintain.